R/shims.R

```diff
@@ -11,7 +11,7 @@
 read.csv <- function(file, ...) {
   if (interactive_log_on()) {
     log_push(file, "utils::read.csv")
-    check_path_safe(file)
+    check_path_safe(file, ... = "utils::read.csv")
```

Comment:
Won't this break if the user wants to pass any options? 

If users can't pass options the package will be debilitating.

Reply:
I think we need to *append* this as a named options. Something like

`... = list(...., func = "utils::read.csv")`

??

Reply:
Can you clarify what you mean by this? `check_path_safe()` is only looking at the path that the user passed and doesn't care about any other arguments/options. The dots there are just a way of telling `check_path_safe()` what to return in its error message. The actual execution of the original function still works like normal since I didn't change that line--which is the one below the `check_path_safe()` call--at all. When I run `fertile::read.csv()` with options other than path (when I've given it a relative path) it executes the function just fine and doesn't break.

Reply:
Oh sorry. I misread the code and thought you were passing this in the next line.